src/activities/hoc2019/PondCreator.jsx

```diff
@@ -69,6 +71,23 @@ export default class PondCreator extends React.Component {
     }
   };
 
+  loadTraining() {
+    const storedTraining = localStorage.getItem(SESSION_KEY);
+    if (storedTraining) {
+      this.state.trainer.loadDatasetJSON(storedTraining);
+    }
+  }
+
+  saveTraining = () => {
+    localStorage.setItem(SESSION_KEY, this.state.trainer.getDatasetJSON());
```

Comment:
I'm not sure if it's better to use `localStorage` or `sessionStorage`. Would welcome input!

Reply:
i think localStorage sounds like the right solution at this point. i like that localStorage persists for longer, but that may end up causing annoying issues where we need to clear localStorage more often for signed out users.

i think for now, let's go with localStorage, and we can easily swap it out once we have the whole flow a little more nailed down

Reply:
Yeah that was my thoughts too. My thought was that local storage is more likely to have unexpected persisted data but that's a product question and we could add some messaging around it or something.